models/repository/repository.go

```diff
@@ -15,14 +20,24 @@ type RepositoryStruct struct {
 	RepositoryID int64
 	Owner        sql.NullString
 	Name         sql.NullString
+	WebhookKey   string
 	database     db.DB
 }
 
-func NewRepository(id int64, repositoryID int64, owner string, name string) *RepositoryStruct {
+func GenerateWebhookKey(seed string) string {
```

Comment:
__[golint]__
* [golint] exported function GenerateWebhookKey should have comment or be unexported